### ISSUE TITLE
Create scheduler plugin.

### ISSUE THREAD

**Issue by husky-parul** [sustainable-computing-io/peaks#9]
Dependent on 
- https://github.com/sustainable-computing-io/peaks/issues/3
- https://github.com/sustainable-computing-io/peaks/issues/4
- https://github.com/sustainable-computing-io/peaks/issues/5
- https://github.com/sustainable-computing-io/peaks/issues/6
- https://github.com/sustainable-computing-io/peaks/issues/7

**Comment by vsoch** [comment]
Hi! Is work still underway here? I am interested in the project idea but I don't see any custom scheduler plugin code (is it somewhere else)? Thanks!

I just watched your Kubecon talk about Kelper, new to this space and wanted to say the idea for using power models is really neat!

**Comment by linericyang** [comment]
You can refer to [repo](https://github.com/sustainable-computing-io/scheduler-plugins/tree/master/pkg/peaks) for peaks scheduler plugin and deployment instructions.

**Comment by vsoch** [comment]
Perfect, thank you! Is there a link to that somewhere (prominently) here?

**Comment by linericyang** [comment]
I don't believe such a link exists yet, but it should include one to demonstrate how to utilize a trained model within a scheduler and deploy it in Kubernetes.
